src/KilnBase.sol

```diff
@@ -57,67 +48,75 @@ abstract contract DssKiln {
     }
 
     modifier auth {
-        require(wards[msg.sender] == 1, "DssKiln/not-authorized");
+        require(wards[msg.sender] == 1, "KilnBase/not-authorized");
         _;
     }
 
-    // --- Mutex  ---
     modifier lock {
-        require(locked == 0, "DssKiln/system-locked");
+        require(locked == 0, "KilnBase/system-locked");
         locked = 1;
         _;
         locked = 0;
     }
 
-    function _min(uint256 x, uint256 y) internal pure returns (uint z) {
+    function _min(uint256 x, uint256 y) internal pure returns (uint256 z) {
         return x <= y ? x : y;
```

Comment:
```suggestion
        z = x <= y ? x : y;
```

Reply:
mm yeah this varies between some of the Maker contracts, but there are at least a bunch of contracts using the current code so I think we can leave as is:

* https://github.com/makerdao/dss-auto-line/blob/65c3da68d3ad63aebdd86cc669739da594f0f55c/src/DssAutoLine.sol
* https://github.com/makerdao/dss/blob/17187f7d47be2f4c71d218785e1155474bbafe8a/src/vow.sol#L91
* https://github.com/makerdao/dss/blob/fa4f6630afb0624d04a003e920b0d71a00331d98/src/end.sol#L302

Reply:
Sure, no issue here then.